ui_components/src/village/WarTable.js

```diff
@@ -17,6 +16,194 @@ export function WarTable({
+                    </div>
+                </div>
+            </div>
+            <div className="row second">
+                <div className="column first">
+                    <div className="header">global war score</div>
+                    <div className="global_leaderboard_container">
+                        <div className="warlog_label_row">
+                            <div className="warlog_username_label"></div>
+                            <div className="warlog_war_score_label">war score</div>
+                            <div className="warlog_pvp_wins_label">pvp wins</div>
+                            <div className="warlog_raid_label">raid</div>
+                            <div className="warlog_reinforce_label">reinforce</div>
+                            <div className="warlog_infiltrate_label">infiltrate</div>
+                            <div className="warlog_defense_label">def</div>
+                            <div className="warlog_captures_label">captures</div>
+                            <div className="warlog_patrols_label">patrols</div>
+                            <div className="warlog_resources_label">resources</div>
+                            <div className="warlog_chart_label"></div>
+                        </div>
+                        {globalLeaderboardWarLogs
+                            .map((log, index) => (
+                                <PlayerWarLog log={log} index={index} animate={true} getVillageIcon={getVillageIcon} />
+                            ))}
+                    </div>
+                    <div className="global_leaderboard_navigation">
+                        <div className="global_leaderboard_navigation_divider_left">
+                            <svg width="100%" height="2"><line x1="0%" y1="1" x2="100%" y2="1" stroke="#4e4535" strokeWidth="1"></line></svg>
+                        </div>
+                        <div className="global_leaderboard_pagination_wrapper">
+                            {globalLeaderboardPageNumber > 1 && <a className="global_leaderboard_pagination" onClick={() => GlobalLeaderboardPreviousPage(globalLeaderboardPageNumber - 1)}>{"<< Prev"}</a>}
+                        </div>
+                        <div className="global_leaderboard_navigation_divider_middle"><svg width="100%" height="2"><line x1="0%" y1="1" x2="100%" y2="1" stroke="#4e4535" strokeWidth="1"></line></svg></div>
+                        <div className="global_leaderboard_pagination_wrapper">
+                            <a className="global_leaderboard_pagination" onClick={() => GlobalLeaderboardNextPage(globalLeaderboardPageNumber + 1)}>{"Next >>"}</a>
+                        </div>
+                        <div className="global_leaderboard_navigation_divider_right"><svg width="100%" height="2"><line x1="0%" y1="1" x2="100%" y2="1" stroke="#4e4535" strokeWidth="1"></line></svg></div>
+                    </div>
```

Comment:
This is probably also a good candidate to be its own component. Maybe it could be re-used for war records navigation, maybe not